Dockerfile.jupyterlab

```diff
@@ -60,6 +60,11 @@ COPY scripts/install-code-server.sh /opt/scripts/install-code-server.sh
 COPY jupyterlab /opt/jupyterlab
 RUN /opt/jupyterlab/postBuild
 
+# ========== pyvista setup ============
+# allow jupyterlab for ipyvtk
+ENV JUPYTER_ENABLE_LAB=yes
+ENV PYVISTA_TRAME_SERVER_PROXY_PREFIX='/proxy/'
+
```

Comment:
I would remove these since `PYVISTA_TRAME_SERVER_PROXY_PREFIX` has to be set at runtime for JupyterHub deployments

```suggestion
```